frontend/src/routes/Infrastructure/Clusters/ManagedClusters/ImportCluster/ImportCluster.tsx

```diff
@@ -215,6 +220,19 @@ export default function ImportClusterPage() {
     const [submitButtonText, setSubmitButtonText] = useState<string>()
     const [submittingButtonText, setSubmittingButtonText] = useState<string>()
     const [state, dispatch] = useReducer(reducer, getInitialState(initialClusterName, initialServer))
+    const [configMaps] = useRecoilState(configMapsState)
+    const [subscriptionOperators] = useRecoilState(subscriptionOperatorsState)
```

Comment:
If you're not using the setter for the state, you can use `useRecoilValue`.
```suggestion
    const configMaps]= useRecoilValue(configMapsState)
    const subscriptionOperators = useRecoilValue(subscriptionOperatorsState)
```